internal/tls/tls_test.go

```diff
@@ -128,36 +231,80 @@ func TestNewClientConfig(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns cfg and nil when option is empty",
+			want: want{
+				err: nil,
+			},
+			checkFunc: func(w want, c *Config, err error) error {
+				if !errors.Is(err, w.err) {
+					return fmt.Errorf("got error = %v, wantErr %v", err, w.err)
+				}
+				if c == nil {
+					return errors.New("config is nil")
+				}
+				return nil
+			},
+		},
+		{
+			name: "returns cfg and nil when cert and key option is not empty",
+			args: args{
+				opts: []Option{
+					WithCert("./testdata/dummyServer.crt"),
+					WithKey("./testdata/dummyServer.key"),
+				},
+			},
+			want: want{
```

Comment:
```suggestion
```
default value of error is nil, so could you please delete?

Reply:
sometimes i was intentionally add this line because I think it is more clear to declare this instead of nothing.
what do you think?

Reply:
I think I've come to the conclusion that I don't write anything with default values ​​before.

Reply:
yes... seems it is meaningless but I think it is easier to read... but this time I think we can remove it.